src/status_im/navigation/events.cljs

```diff
@@ -146,6 +146,10 @@
 
 (rf/reg-event-fx :open-share open-share)
 
+(rf/reg-event-fx :open-url
+ (fn [_ [url]]
+   {:fx [[:open-url url]]}))
```

Comment:
@seanstrom, our guideline is to name effects prefixed by `:effects`, like `:effects/open-url`. Could you rename the effect in this PR?

Reply:
Yup sounds good, I've added commit for renaming the effect 👍